scripts/us_census/geojsons_low_res/download.py

```diff
@@ -90,22 +94,32 @@ def download_data(self, place='country/USA'):
         # to get the 0th element explicitly.
         assert len(geolevel[place]) == 1
         geolevel = geolevel[place][0]
+
+        level_to_fetch = geolevel
+        for i in range(level):
+            if level_to_fetch not in self.LEVEL_MAP:
+                raise ValueError("Desired level does not exist.")
+            level_to_fetch = self.LEVEL_MAP[level_to_fetch]
+
```

Comment:
Can we eliminate these two lines, directly using the value?

Reply:
See comment above.